packages/x-date-pickers/src/PickersShortcuts/PickersShortcuts.tsx

```diff
@@ -85,7 +85,12 @@ function PickersShortcuts<TValue>(props: PickersShortcutsProps<TValue>) {
         {
           maxHeight: VIEW_HEIGHT,
           maxWidth: 200,
-          overflow: 'auto',
+          scrollbarGutter: 'stable',
+          width: '100%',
+          overflow: 'hidden',
```

Comment:
IMHO, the experience on mobile is not great. 🙈 
Could we conditionally apply these styles only when not on mobile?
AFAIK, mobiles usually if not always have similar scrollbar handling to Safari/MacOS.

Reply:
Done in [8469175](https://github.com/mui/mui-x/pull/13972/commits/846917521dfdc9eae42f0a9decab573dc9794f2e)